modules/platforms/cpp/thin-client-test/src/ignite_client_test.cpp

```diff
@@ -71,14 +72,52 @@ class IgniteClientTestSuiteFixture
 
         BOOST_CHECK_EQUAL(GetActiveConnections(), expect);
     }
+    /**
+     * Check that client started with specified size of user thread pool started exactly the specified number of threads
+     * in thread pool.
+     *
+     * @param cfg Client configuration.
+     * @param num Expected thread number
+     */
+    static void CheckThreadsNum(IgniteClientConfiguration &cfg, uint32_t num)
+    {
+        ignite::TestServer server;
+        server.PushHandshakeResponse(true);
+        server.Start();
+
+        int32_t threadsBefore = ignite::common::concurrent::GetThreadsCount();
```

Comment:
`GetThreadsCount` is used only for tests, should we keep it in the `common` module?